### PR TITLE
WIP: Centralize stage activity creation

### PR DESCRIPTION
**Pull request by vsp-gleich**

### CODE REVIEW

**Comment by vsp-gleich** [comment]
finish centralizing stage activity creation (issue #917 )

File: matsim/src/main/java/org/matsim/core/population/PopulationUtils.java [hunk 1]
```diff
@@ -833,8 +833,8 @@ public static Activity createAndAddActivity(Plan plan, String type) {
 		return act ;
 	}
 
-	public static Activity createStageActivityFromCoordLinkIdAndMode( final Coord interactionCoord, final Id<Link> interactionLink, String mode ) {
-		Activity act = createActivityFromCoordAndLinkId(PlanCalcScoreConfigGroup.createStageActivityType(mode), interactionCoord, interactionLink);
+	public static Activity createStageActivityFromCoordLinkIdAndModePrefix(final Coord interactionCoord, final Id<Link> interactionLink, String modePrefix ) {
+		Activity act = createActivityFromCoordAndLinkId(PlanCalcScoreConfigGroup.createStageActivityType(modePrefix), interactionCoord, interactionLink);
```
Comment (michalmac) [inline matsim/src/main/java/org/matsim/core/population/PopulationUtils.java]: Could stage activities created here be of a different Activity (sub)class that fails on setting start/end time? (e.g. throwing an `UnsupportedOperationException` ?)

Comment (vsp-gleich) [reply matsim/src/main/java/org/matsim/core/population/PopulationUtils.java]: This is already planned (however, without a clear time horizon):
[https://matsim.atlassian.net/projects/MATSIM/issues/MATSIM-847](https://matsim.atlassian.net/projects/MATSIM/issues/MATSIM-847)

Comment (vsp-gleich) [reply matsim/src/main/java/org/matsim/core/population/PopulationUtils.java]: I see this pull request as another step into the direction of introducing a PlanElement `Waypoint`

**Comment by vsp-gleich** [comment]
All stage activities are now created using PopulationUtils.createStageActivityFromCoordLinkIdAndMode except contrib socnetsim with its special interaction activities (which operate on a different level than the other stage activities and for that reason do not end on " interaction" like all the others).

**Pull request merged**